src/main/java/frc/robot/RobotContainer.java

```diff
@@ -25,13 +26,16 @@ public RobotContainer() {
     configureBindings();
 
     m_shoot.setDefaultCommand(m_shoot.stopMotorCommand());
```

Comment:
I see what you're going for here, and this is fine for now. There is, IMO, a slightly better way to recreate this behavior (stopping the motor when it's not in use), but the explanation is a bit wordy so we can go over that on Saturday.

Reply:
👍